app/screens/selector_screen/selector_screen.js

```diff
@@ -322,7 +325,7 @@ export default class SelectorScreen extends PureComponent {
         return (
             <View style={style.container}>
                 <StatusBar/>
-                <View style={style.searchBar}>
+                <View style={[style.searchBar, padding(true)]}>
```

Comment:
why true here?

Reply:
This was my mistake, thanks for catching it @enahum